field/bitmap.go

```diff
@@ -129,12 +134,12 @@ func (f *Bitmap) Unmarshal(v interface{}) error {
 	return nil
 }
 
-func (f *Bitmap) SetData(data interface{}) error {
-	if data == nil {
+func (f *Bitmap) Marshal(v interface{}) error {
+	if v == nil {
 		return nil
 	}
 
-	bmap, ok := data.(*Bitmap)
+	bmap, ok := v.(*Bitmap)
```

Comment:
In this field, we need `data` as it's basically `value` - it stores the value of the field. Please, don't be confused with my previous statement that we should remove `data` from all fields. When `data *String` (or pointer to other field type - then we should remove it), but here - `data` has the same purpose as `value` in other fields. It just has different name.